### ISSUE TITLE
Wrong width/height for jpg ( ST3 )

### ISSUE THREAD

**Issue by pustovalov** [liamcain/AutoFileName#37]
HTML:
![1](https://f.cloud.github.com/assets/1568885/962175/2f9872a0-04eb-11e3-815d-5755c25f5caf.png)

when you choose any jpeg, you get:

``` javascript
<img src="images/bg_header.jpg" height="-1" width="-1" alt="">
```

CSS:
![2](https://f.cloud.github.com/assets/1568885/962199/b4dc5f30-04eb-11e3-99e5-2ddad8400f4e.jpg)

For css show width/height may not be important but it is very convenient, you can see from ST what size image.

**Comment by liamcain** [comment]
This is a known issue. When Sublime Text made the switch from Python 2 to Python 3, the script I was using to get the image dimensions broke. I was able to fix it for pngs and gifs but jpgs remain broken. For anyone who wants to help, the issue is [commented out here](https://github.com/BoundInCode/AutoFileName/blob/st3/getimageinfo.py#L37).

**Comment by matthacksteiner** [comment]
any news on this? thanks...

**Comment by zslabs** [comment]
Really missin this in the ST3 version - wish I could help troubleshoot, but Python isn't my thang ;)

**Comment by manafire** [comment]
Added JPG dimensioning: https://github.com/BoundInCode/AutoFileName/pull/47

**Comment by liamcain** [comment]
Fixed in #47.
